pulpcore/cli/rpm/context.py

```diff
@@ -195,5 +195,12 @@ class PulpRpmRepositoryContext(PulpRepositoryContext):
     VERSION_CONTEXT = PulpRpmRepositoryVersionContext
     CAPABILITIES = {"sync": [PluginRequirement("rpm")], "pulpexport": [PluginRequirement("rpm")]}
 
+    def sync(self, href: Optional[str] = None, body: Optional[EntityDefinition] = None) -> Any:
```

Comment:
@mdellweg  - Is this what you meant/intended by "expected this in the context" comment?